testsuite/rust_generator/tests/AC/test.yml

```diff
@@ -0,0 +1,5 @@
+language: PY3
+time: 2
+memory: 65536
+source: sol.py
+expect: [AC, AC, AC]
```

Comment:
```suggestion
expect: AC
```

Reply:
Also updated the test I copied this from, lol.